src/main/scala/is/hail/variant/Genotype.scala

```diff
@@ -78,21 +78,23 @@ abstract class Genotype extends Serializable {
     else
```

Comment:
(not a comment on this line)
replace isDosage with isLinearScaled or something.